lib/irb/cmd/pa.rb

```diff
@@ -19,7 +19,7 @@ def execute(expression)
     end
   end
 
-  if PowerAssert.newer_irb?
+  if ::IRB::Command.respond_to?(:register)
```

Comment:
https://github.com/ruby/irb/commit/462c1284af0e6d4c4177a6dd30ed3dfc84f303db

::IRB::Command born in v1.12.0 🤔